src/main/java/pl/simpleascoding/tutoringplatform/domain/advertisement/Advertisement.java

```diff
@@ -0,0 +1,49 @@
+package pl.simpleascoding.tutoringplatform.domain.advertisement;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.simpleascoding.tutoringplatform.domain.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Entity
+@Table(name = "advertisement")
+@NoArgsConstructor
+public class Advertisement {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
You can use `long` for your `id` field instead of `Long`, but don't change it in the repository or it will break

Reply:
RIght, but most of other entities has also `Long` instead `long`, so I think that it should be consistent